randovania/games/super_metroid/json_data/Crateria.txt

```diff
@@ -177,17 +177,31 @@ Extra - asset_id: 4
   * Normal Door to The Moat/Door to West Ocean
   > Door to Wrecked Ship Entrance
       Any of the following:
-          # TODO: Check to see if possible with HBJ.
           Grapple Beam or Space Jump
-          # It is possible to jump across by not using full jumps to avoid bonking on the ceiling.
+          Horizontal Infinite Bomb Jump (Expert) and Can Lay Bombs
+          # It is possible to jump across by not using full jumps to avoid bonking on the celing.
           Movement (Intermediate)
           All of the following:
-              # From the sea floor
+              # From the Sea Floor
               Gravity Suit
               Any of the following:
                   Hi-Jump Boots or Wall Jumps (Beginner)
                   # IBJ
                   Vertical Infinite Bomb Jump (Beginner) and Can Lay Bombs
+  > Door from Gravity Suit Room
+      Any of the following:
+          Space Jump
+          All of the following:
+              # Get over and WJ up
+              Wall Jumps (Intermediate)
+              Grapple Beam or Movement (Intermediate)
+          All of the following:
+              # From the sea floor
+              Gravity Suit
+              Any of the following:
+                  Vertical Infinite Bomb Jump (Advanced) and Can Lay Bombs
+                  # Spark from the sea floor
+                  Speed Booster and Shinespark (Beginner) and Shinespark Damage ≥ 51
```

Comment:
So this is strange.
In the editor I see this, which i wanted to comment on that it needs the "can lay bombs" template and a short comment.
![grafik](https://github.com/randovania/randovania/assets/38186597/eeba4117-c384-4a1d-947e-865b5dd36ae9)


but i cant actually see the change in here. that or i'm blind.

Reply:
I can see it now, just needed a proper reload i guess.

Reply:
Added.